requirements.txt

```diff
@@ -5,5 +5,6 @@ datefinder~=0.7
 stix2~=2.1.0
 pika~=1.2
 sseclient~=0.0.27
+python_json_logger==2.0.2
```

Comment:
Please change that to `python_json_logger~=2.0.2`

Reply:
Thank you for your feedbacks @nor3th, I have made the changes. 

However, I was not able to run the tests: `pytest -k test_internal_enrichment_connector --connectors -vvl`:
`tests/02-integration/connector/test_connector.py::test_internal_enrichment_connector[ipv4_enrichment]` hangs indefinitely, the message created by the connector is never consumed in RabbitMQ, do you have any idea why? It works fine with real connectors. 

I will do the appropriate changes in the worker as well (passing the `json_logging` variable to the constructor of the`OpenCTIApiClient` used by the worker).

Reply:
Thank you for the changes. I'll have a look at that the test case in the upcoming days if that's fine with you?

Reply:
Sure, thank you!

Reply:
The tests are working, it was an error with my local OpenCTI configuration.

Reply:
Sounds great, thank you for your effort!